Adding Android SDK Tools info (#1067)
apache/cordova-docs: www/docs/en/dev/guide/platforms/android/index.md

1. Android SDK build-tools version 19.1.0 or higher
1. Android Support Repository (found under the "SDK Tools" tab)

> #### Android SDK Tools: 

Reviewer: ```suggestion
#### Android SDK Tools:
```

Minor trailing space removed.

In Android Studio 3.6 or later, you need to manually add the old version of the Android SDK Tools. To do this:

> 1. Open the Android Studio SDK Manager

Reviewer 2: ```suggestion
1. Open the Android Studio **SDK Manager**
```

> 2. In the Android SDK tab, uncheck `Hide Obsolete Packages`

Reviewer 2: ```suggestion
2. In the Android **SDK Tools** tab, uncheck `Hide Obsolete Packages`
```

> 2. In the Android SDK tab, uncheck 'Hide Obsolete Packages'

Reviewer: ```suggestion
2. In the Android SDK tab, uncheck `Hide Obsolete Packages`
```

> 3. Check Android SDK Tools (Obsolete)

Reviewer 2: ```suggestion
3. Check `Android SDK Tools (Obsolete)`
```
